ci/dev/test.sh

```diff
@@ -9,7 +9,8 @@ main() {
   # information. We must also run it from the root otherwise coverage will not
   # include our source files.
   cd "$OLDPWD"
-  ./test/node_modules/.bin/jest "$@"
+  # We use the same environment variables set in ci.yml in the test job
+  PASSWORD=e45432jklfdsab CODE_SERVER_ADDRESS=http://localhost:8080 ./test/node_modules/.bin/jest "$@"
```

Comment:
We don't need to set these because they're already set from CI right? And locally we wouldn't want to set these either I think since devs might have their code-server on a different port or with a different password.

Reply:
If they aren't set we'll probably want to error so the dev knows what to set to make the tests work.

Reply:
> And locally we wouldn't want to set these either I think since devs might have their code-server on a different port or with a different password.

Ah good point. I didn't think of that.

Reply:
I'll modify that and add print an error message telling them to set those.

Eventually when e2e is separate, it will be more clear.

Reply:
<img width="535" alt="image" src="https://user-images.githubusercontent.com/3806031/106936166-79a93200-66d9-11eb-8437-cc6c309a672d.png">